### PR TITLE
update atomicdex sidebar menu

### PR DESCRIPTION
**Pull request by smk762**
closes https://github.com/KomodoPlatform/developer-docs/issues/380

### CODE REVIEW

**Review by gcharang — APPROVED** [review]

File: docs/.vuepress/atomicDEX-sidebar.js [hunk 1]
```diff
@@ -115,130 +115,102 @@ let atomicDEXsidebar = {
...
+          children: [
+          ["/basic-docs/atomicdex-api-legacy/best_orders.md", "best_orders"],
+          ["/basic-docs/atomicdex-api-legacy/buy.md", "buy"],
+          ["/basic-docs/atomicdex-api-legacy/cancel_all_orders.md", "cancel_all_orders"],
+          ["/basic-docs/atomicdex-api-legacy/cancel_order.md", "cancel_order"],
+          ["/basic-docs/atomicdex-api-legacy/orderbook.md", "orderbook"],
+          ["/basic-docs/atomicdex-api-legacy/orderbook_depth.md", "orderbook_depth"],
+          ["/basic-docs/atomicdex-api-legacy/orders_history_by_filter.md", "orders_history_by_filter"],
+          ["/basic-docs/atomicdex-api-legacy/order_status.md", "order_status"],
+          ["/basic-docs/atomicdex-api-legacy/my_orders.md", "my_orders"],
```
Comment (artemii235) [inline docs/.vuepress/atomicDEX-sidebar.js:165]: `my_orders` should be right after `cancel_order` :)

File: docs/.vuepress/atomicDEX-sidebar.js [hunk 2]
```diff
@@ -282,95 +269,113 @@ let atomicDEXsidebar = {
...
+              "/basic-docs/atomicdex-api-20-dev/remove_delegation.md",
+              "remove_delegation",
+            ]
+          ]
+        },
+        {
+          title: "Utility",
+          collapsable: true,
+          children: [
+            ["/basic-docs/atomicdex-api-20-dev/recreate_swap_data.md", "recreate_swap_data"],
```
Comment (artemii235) [inline docs/.vuepress/atomicDEX-sidebar.js:364]: I think these can be moved to `Swaps` section.

**Review by artemii235 — CHANGES_REQUESTED** [review]
Thanks for the PR! A couple of notes.

File: docs/.vuepress/atomicDEX-sidebar.js (hunk 1 above)
Comment (smk762) [reply docs/.vuepress/atomicDEX-sidebar.js:165]: :facepalm: I need to go back to kindergarten :sweat_smile:

File: docs/.vuepress/atomicDEX-sidebar.js (hunk 2 above)
Comment (smk762) [reply docs/.vuepress/atomicDEX-sidebar.js:364]: There was no "swaps" subsection in the 2.0 sections, so I renamed "Utility" to "Swaps". In the legacy section, I also moved swaps related methods from utility subsection to swaps subsection.

**Comment by smk762** [comment]
@artemii235 I've also added a table of available methods to this PR
https://github.com/KomodoPlatform/developer-docs/blob/66b000ef06b2476c70e28d20fc275c315fe97cfd/docs/basic-docs/atomicdex/atomicdex-methods.md

Initial intent was to track what is missing or needs to move to v2 in Desktop GUI, though I figured it might be a good reference for others also.

With the latest release, I'll need to review what has moved from dev into master and update the table/sidebar accordingly.

**Comment by smk762** [comment]
I've moved all methods now in master to their appropriate position within the sidebar menus and table, and updated urls within pointing to the older reference doc locations.

**Review by artemii235 — APPROVED** [review]
👍

**Pull request merged**